workflow-service/src/main/java/com/redhat/parodos/project/repository/ProjectUserRoleRepository.java

```diff
@@ -13,4 +13,6 @@ public interface ProjectUserRoleRepository extends JpaRepository<ProjectUserRole
 
 	List<ProjectUserRole> findByUserId(UUID userId);
 
+	List<ProjectUserRole> deleteAllByIdProjectIdAndIdUserIdIn(UUID projectId, List<UUID> userIds);
```

Comment:
there is no coverage for the new *Repository classes introduced in the previous PR. pls add.